mlrun/runtimes/local.py

```diff
@@ -481,9 +488,17 @@ def get_func_arg(handler, runobj: RunObject, context: MLClientCtx, is_nuclio=Fal
         elif key in params:
             kwargs[key] = copy(params[key])
         elif key in inputs:
-            obj = context.get_input(key, inputs[key])
-            if type(args[key].default) is str or args[key].annotation == str:
-                kwargs[key] = obj.local()
-            else:
-                kwargs[key] = context.get_input(key, inputs[key])
+            kwargs[key] = _get_input_value(key)
+
+    # VAR_KEYWORD meaning : A dict of keyword arguments that aren’t bound to any other parameter.
+    # This corresponds to a **kwargs parameter in a Python function definition.
+    if any(param.kind == param.VAR_KEYWORD for param in args.values()):
```

Comment:
`args = inspect.signature(handler).parameters` means that `args` is an `OrderedDict` and because `**kwargs` can only be last in the function's parameters list, you can simply check the last one:

```suggestion
    if list(args.values())[-1].kind == param.VAR_KEYWORD:
```